app/pages/sections/catalogue-solutions/routes.js

```diff
@@ -93,8 +95,19 @@ export const catalogueSolutionsRoutes = (authProvider, addContext, sessionManage
 
   router.get('/select-solution/select-price/select-recipient', authProvider.authorise({ claim: 'ordering' }), withCatch(authProvider, async (req, res) => {
     const { orderId } = req.params;
+    const accessToken = extractAccessToken({ req, tokenType: 'access' });
+    const solutionId = sessionManager.getFromSession({ req, key: 'selectedSolution' });
```

Comment:
we should probably call the key in session `selectedSolutionId` if it is only storing the id